projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts

```diff
@@ -1310,121 +1310,121 @@ export const GENERATED_REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
+      "Class SaveCartAdapter has been removed and is no longer part of the public API. Use CartAdapter.save instead from '@spartacus/cart/base/core'",
   },
   {
     node: 'SaveCartConnector',
     importPath: '@spartacus/core',
     comment:
-      'Class SaveCartConnector has been removed and is no longer part of the public API. ',
+      "Class SaveCartConnector has been removed and is no longer part of the public API. Use CartConnector.save instead from '@spartacus/cart/base/core'",
   },
   {
     node: 'USER_ORDER_DETAILS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_ORDER_DETAILS has been removed and is no longer part of the public API. ',
+      "Variable USER_ORDER_DETAILS has been removed and is no longer part of the public API. It is replaced by ORDER_DETAILS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_ORDERS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_ORDERS has been removed and is no longer part of the public API. ',
+      "Variable USER_ORDERS has been removed and is no longer part of the public API. It is replaced by ORDERS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_REPLENISHMENT_ORDER_DETAILS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_REPLENISHMENT_ORDER_DETAILS has been removed and is no longer part of the public API. ',
+      "Variable USER_REPLENISHMENT_ORDER_DETAILS has been removed and is no longer part of the public API. It is replaced by REPLENISHMENT_ORDER_DETAILS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_REPLENISHMENT_ORDERS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_REPLENISHMENT_ORDERS has been removed and is no longer part of the public API. ',
+      "Variable USER_REPLENISHMENT_ORDERS has been removed and is no longer part of the public API. It is replaced by REPLENISHMENT_ORDERS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_RETURN_REQUEST_DETAILS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_RETURN_REQUEST_DETAILS has been removed and is no longer part of the public API. ',
+      "Variable USER_RETURN_REQUEST_DETAILS has been removed and is no longer part of the public API. It is replaced by RETURN_REQUEST_DETAILS from '@spartacus/order/core', but the constant value is not the same.",
   },
   {
     node: 'USER_RETURN_REQUESTS',
     importPath: '@spartacus/core',
     comment:
-      'Variable USER_RETURN_REQUESTS has been removed and is no longer part of the public API. ',
+      "Variable USER_RETURN_REQUESTS has been removed and is no longer part of the public API. It is replaced by RETURN_REQUESTS from '@spartacus/order/core', but the constant value is not the same.",
```

Comment:
![Screen Shot 2022-07-20 at 4 05 36 PM](https://user-images.githubusercontent.com/44440575/180062281-ee1bf660-204d-4ee5-b045-ea322e5679d9.png)

`USER_RETURN_REQUESTS` is from `order-state.ts`? It is not there.